php/class-fieldmanager-colorpicker.php

```diff
@@ -43,6 +50,12 @@ public function __construct( $label = '', $options = array() ) {
 
 		$this->sanitize = array( $this, 'sanitize_hex_color' );
 
+		if ( ! empty( $options['default_color'] ) ) {
```

Comment:
My thinking was that since there is no real reason for a `''` default_color value, we should instead check for `! empty()` and not `isset()` to account for that.  In other words, when the user sets a default_color of `''` then we should change it to the default_value.  I hope that makes sense 😄